Makefile.js

```diff
@@ -562,19 +562,6 @@ target.karma = () => {
 
     target.webpack("production");
 
-    const browserFileLintOutput = new CLIEngine({
```

Comment:
Why are we removing this step?

Reply:
I'm assuming it's because the new targets allow for some ES6+ features to remain untranspiled, but @mysticatea can confirm.

Reply:
Right. New build result contains ES2015+ syntax, so this check fails 100%.